### PR TITLE
feat: api rename application to program

### PR DESCRIPTION
**Pull request by sunnygleason**
PROBLEM:

* `application` isn't a thing (yet, if ever)

SOLUTION:

* long live `program`!

### CODE REVIEW

**Comment by sunnygleason** [comment]
@manuel-calavera FYI this will hopefully be landing soon, let me know if/how we should order things to avoid merge conflicts...

@CriesofCarrots @mvines FYI this should be ready soonish...

**Comment by manuel-calavera** [comment]
@sunnygleason can we merge my material table pr first?

**Comment by sunnygleason** [comment]
@manuel-calavera I just submitted the review for your other PR - I think there are minimal conflicts (hopefully!), please let me know when it's ready!

File: src/v2/components/Accounts/Detail/Code/index.jsx [hunk 1]
```diff
@@ -9,17 +9,17 @@ import useStyles from './styles';
 
 const AccountCode = ({accountView}: {accountView: Object}) => {
   const classes = useStyles();
-  const applicationCode = YAML.stringify(accountView);
+  const accountCode = YAML.stringify(accountView);
```
Comment (CriesofCarrots) [inline src/v2/components/Accounts/Detail/Code/index.jsx]: 👍

**Pull request merged**